collectionsPrelude.grace

```diff
@@ -1608,6 +1624,15 @@ class dictionary⟦K,T⟧ {
             }
             self
         }
+        method clear {
+            inner := _prelude.primitiveArray.new(8)
+            numBindings := 0
+            mods := 0
```

Comment:
Clear is a modification, like add or delete.  So the right thing is, I think, mods := mods + 1.  mods := 0 will work too, of course — all that matters is that mods is changed.

But this makes me realize that all of the _other_ clear methods here do not change mods, and they must!